### PR TITLE
Gesture handler wheel

### PR DESCRIPTION
**Pull request by kyi193**
Created the base project that renders a dummy page that displays a working wheel spinner with dummy pizza restaurant names.

### CODE REVIEW

File: components/GestureSpinnerWheel.js [hunk 1]
```diff
@@ -1,10 +1,27 @@
 import React, { Component } from 'react'
 import { View, Text, StyleSheet } from 'react-native'
+import WheelOfFortune from 'react-native-wheel-of-fortune'
+
+
```
Comment (squidapillar) [inline components/GestureSpinnerWheel.js]: remove extra newlines

File: components/GestureSpinnerWheel.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ class GestureSpinnerWheel extends Component {
     selectedIndex: null
   }
   render() {
-    const rewards = ['Joe\'s', 'Rays', '2Bros', 'Dominos', '&Pizza', '$1Pizza']
+    const rewards = ['Joes', 'Rays', '2Bros', '&Pizza', '$1Pizza']
```
Comment (squidapillar) [inline components/GestureSpinnerWheel.js:11]: What was the purpose of changing the names of the restaurants here?

Comment (squidapillar) [inline components/GestureSpinnerWheel.js:11]: shouldn't this variable name be `restaurants`, not sure what `rewards` means

File: components/GestureSpinnerWheel.js [hunk 3]
```diff
@@ -2,8 +2,6 @@ import React, { Component } from 'react'
 import { View, Text, StyleSheet } from 'react-native'
 import WheelOfFortune from 'react-native-wheel-of-fortune'
 
-
```
Comment (squidapillar) [inline components/GestureSpinnerWheel.js]: these changes should be squashed into the earlier commits that introduced them in the first place

**Comment by squidapillar** [comment]
@kyi193 looking good so far, just need the final round of rebase master (probably should have a safe branch till you figure it out)

**Comment by kyi193** [comment]
@squidapillar I merged gesture-handler-wheel with the master branch.

File: components/GestureSpinnerWheel.js [hunk 4]
```diff
@@ -1,19 +1,42 @@
...
-        <Text>Hi</Text>
+        <WheelOfFortune
+          onRef={ref => (this.child = ref)}
+          rewards={restaurants}
+          knobSize={20}
+          borderWidth={3}
+          borderColor={"#FFF"}
+          winner={Math.floor(Math.random() * restaurants.length)}
+          innerRadius={10}
+          textColor={`#FFFFFF`}
```
Comment (squidapillar) [inline components/GestureSpinnerWheel.js:31]: nit: don't need string interpolation here, so maybe just stick with `""` quotes

**Pull request merged**